vpn/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -505,6 +505,9 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope(), N
 
             for (service in manager.getRunningServices(Int.MAX_VALUE)) {
                 if (TrackerBlockingVpnService::class.java.name == service.service.className) {
+                    if (Build.VERSION.SDK_INT == Build.VERSION_CODES.M) {
```

Comment:
We should always use `appBuildConfig`
```suggestion
                    if (appBuildConfig.sdkInt == Build.VERSION_CODES.M) {
```

Reply:
Can't use the `appBuildConfig` as `isServiceRunning` is declared in the companion object. I can pass it as param, however this is used in many places.

Reply:
> Can't use the `appBuildConfig` as `isServiceRunning` is declared in the companion object. I can pass it as param, however this is used in many places.

Nah, nvm. Didn't realise it was static call. All good